osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/SystemEventServiceTest.java

```diff
@@ -50,6 +61,8 @@ void setUp() {
             this.osgpRequestMessageSender,
             this.correlationIdProviderService,
             this.invocationCounterEventThreshold);
+    this.device = this.getDevice();
```

Comment:
I don't mind extracting builder invocations into methods, but it does not feel nice when there is general set up, with a general name like `getDevice` that is actually about a specific special case (a device with its invocation counter at the threshold value).